tests/kernels/attention/test_attention_selector.py

```diff
@@ -127,12 +127,13 @@ def test_env(
 
         elif device == "cuda":
             with patch("vllm.platforms.current_platform", CudaPlatform()):
+                capability = torch.cuda.get_device_capability()
```

Comment:
Does this test run on Blackwell in CI? If not, could you add it to Blackwell Tests?